mesheryctl/internal/cli/root/filter/error.go

```diff
@@ -15,30 +15,96 @@
 package filter
 
 import (
+	"fmt"
 	"strconv"
 
 	"github.com/layer5io/meshkit/errors"
 )
 
 const (
-	ErrInvalidAuthTokenCode = "1000"
-	ErrInvalidAPICallCode   = "1001"
-	ErrReadAPIResponseCode  = "1002"
-	ErrUnmarshalCode        = "1003"
+	ErrInvalidAuthTokenCode   = "1000"
+	ErrInvalidAPICallCode     = "1001"
+	ErrReadAPIResponseCode    = "1002"
+	ErrUnmarshalCode          = "1003"
+	ErrMarshalCode            = "1004"
+	ErrMarshalIndentCode      = "1005"
+	ErrReadFileCode           = "1006"
+	ErrReadConfigFileCode     = "1007"
+	ErrProcessConfigFileCode  = "1008"
+	ErrNewRequestCode         = "1009"
+	ErrMakeRequestCode        = "1010"
+	ErrReadResponseBodyCode   = "1011"
+	ErrResponseStatusBodyCode = "1012"
+	ErrResponseStatusCode     = "1013"
+	ErrJSONToYAMLCode         = "1014"
+	ErrOutFormatFlagCode      = "1015"
+	ErrFilterNameOrIDCode     = "1016"
```

Comment:
There are similar errors already present in `mesheryctl/internal/cli/root/errors.go`, use those instead.   I feel most of these errors aren't filter specific. If there's any error absent from the aforementioned file that is not specific to filters add it in that file and use.

Also do not assign specific error codes, replace the error code strings with "replace_me"

Reply:
well from what i see, asides errors for  `filter_ID` and `Filter_name`,  the remaining errors are not filter specific. 
So you want me to write some of this function codes that have not been written initially  in  the `error.go` in the root package instead, so that they can be used globally used by other packages to  ?

Reply:
when i write these functions, i should use the `replace_me` as the error code string

Reply:
`	ErrInvalidAuthTokenCode = "1000"
	ErrInvalidAPICallCode   = "1001"
	ErrReadAPIResponseCode  = "1002"
	ErrUnmarshalCode        = "1003"
	`
	these error codes and the functions were implemented initially by someone else , i think i would leave these ones the way they were , or should i move them to the `root.go`  incase some of the error functions have not been created there initially ? Because some may not be filter specific